src/material-experimental/mdc-core/_core-theme.scss

```diff
@@ -1,8 +1,46 @@
 @use '../../material/core/theming/theming';
+@use '../../material/core/typography/typography';
 @use './option/option-theme';
 @use './option/optgroup-theme';
 @use './elevation';
 
+
+@mixin color($config-or-theme) {
```

Comment:
Is this supposed to be the equivalent of the non-MDC core theme? If yes, it needs some more styles like the ripple and pseudo checkbox.

Reply:
currently we don't have an MDC-based ripple or pseudo checkbox. I think this should be everything for now

Reply:
Couldn't we include the non-MDC ripple here and use this mixin instead of the non-MDC one? Currently we include a bunch of unnecessary styles in the prebuilt themes, because we go through the non-MDC `core` mixin which contains the typographies of all non-MDC components. If you're blocked on this, it could be done in a follow-up.

Reply:
I don't think that would be a good idea, usually people include both the old mixins and new mixins while they're migrating, so if we did that we'd wind up with double the bytes

Reply:
That's basically what we've been doing with the TS code though.